### PR TITLE
image/webp upload type

### PR DESCRIPTION
**Pull request by adevesa**
This changes able to upload images that has image webp encoding.

### CODE REVIEW

File: system/libraries/Upload.php [hunk 1]
```diff
@@ -901,8 +901,7 @@ public function is_allowed_filetype($ignore_mime = FALSE)
 		}
 
 		// Images get some additional checks
-		if (in_array($ext, array('gif', 'jpg', 'jpeg', 'jpe', 'png'), TRUE) && @getimagesize($this->file_temp) === FALSE)
-		{
+		if (in_array($ext, array('gif', 'jpg', 'jpeg', 'jpe', 'png','webp'), TRUE) && @getimagesize($this->file_temp) === FALSE){
```
Comment (narfbg) [inline system/libraries/Upload.php]: Breaking compliance with our [styleguide](https://codeigniter.com/user_guide/general/styleguide.html).

**Comment by narfbg** [comment]
In addition to the inline comment I left, please either incorporate this entire PR in #5883 or separate the config/mimes.php changes in another pull request (you're making changes to it in both PRs - a recipe for merge conflicts).

**Comment by adevesa** [comment]
Fixed compliance, fixed MIMEs

**Comment by narfbg** [comment]
It's still not compliant with our styleguide, curly braces are always on their own line.

**Comment by adevesa** [comment]
Fix compliant

**Pull request merged**